app/scripts/components/dashboardComponents/listItems.jsx

```diff
@@ -26,7 +26,11 @@ import {Table, TableBody, TableHeader, TableHeaderColumn, TableRow, TableRowColu
 class ListItems extends React.Component {
 
     render() {
-        const { allItemsSelected, filesChecked, foldersChecked, isSafari, listItems, loading, nextPage, projectRole, screenSize, tableBodyRenderKey, totalItems, uploads } = mainStore;
+        const {
```

Comment:
Just a small nitpick, when destructuring it's nice to either choose to maintain one line else put each item on it's own line (the same way you would if declaring an object with properties). To be fair, I use to keep them all on one line but have started to place each on it's own line for readability. I'm assuming this is just your editor formatting it this way.

If using multiline, format like: 

```
const { 
    allItemsSelected,
    filesChecked, 
    foldersChecked,
    isSafari 
} = mainStore;
```

if only one item:

```const { foo } = bazStore;```

Reply:
Fixed this for the Navagator ListItems